assets/scss/gravity-forms.scss

```diff
@@ -555,3 +555,8 @@
 	}
   }
 }
+@media only screen and (max-width: 641px) {
+	.gform_legacy_markup_wrapper input:not([type="radio"]):not([type="checkbox"]):not([type="image"]):not([type="file"]) {
+		line-height: 28px !important;
```

Comment:
@todo remove !important if at all possible.

Reply:
Added a todo to look in later. Thanks